ios/FlankerNativeComponents/GameManager.swift

```diff
@@ -333,27 +369,33 @@ class GameManager {
       delegate?.updateText(text: text, color: .black, font: Constants.bigFont, isStart: true, typeTime: .trial)
     }
 
+    setEndTimeViewingImage(time: CACurrentMediaTime(), isStart: true, type: .trial)
```

Comment:
`setText` is also fired when fixations are displayed, can that lead to initializing incorrect values for `startTrialTimestamp` when it gets called from a different place?